api/namex/models/request.py

```diff
@@ -331,11 +331,14 @@ def find_existing_name_by_user(cls, user_name_search_string, email):
         """	
         Gets requests submited by user with given name choice in state draft
         """
+        current_time = datetime.utcnow()
         existing_nr = db.session.query(Request). \
             join(Applicant, and_(Applicant.nrId == Request.id)). \
             filter(
                 Applicant.emailAddress == email,
                 (Request.stateCd == 'DRAFT') | (Request.stateCd == 'PENDING_PAYMENT') ,
+                #check if the request is being made within 5 minutes, if time >= 5 minutes, it will let the user create a new request with same name otherwise it will give the error message to prevent accidental duplicates and accidental double payments for them.
+                Request.submittedDate >= current_time - timedelta(minutes=5),
```

Comment:
i am curious, why is it 5 minutes exactly and not 1 or 10?

Reply:
they want to allow for users that error out to resend the exact same name, after a 5 minute window.  Waiting on Joshua to approve this 5 minutes.   we may lower the time limit.  we just dont want users to DOUBLE Process the exact same Name Request

Reply:
my question is current_time is being set in the same routine that compares the submitted_date.  i think this will always be false.

I think we need an array stored somewhere

submittedDate, when Request to create a DRAFT NR is processed
Applicant_Name
Name Choice 1
Name Choice 2
Name Choice 3.

and a timer after 5 minutes to reset these stored values.

your code should compare the request info with what is stored.

if you are achieving this in the above code, awesome, but i just dont see that.

Reply:
Our data shows many duplicate NR submissions within 1-2 minutes. We're making a change to fix duplicates without blocking others. A 5-minute window seems reasonable to us.